hack/verify-versions.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT="$(dirname "${BASH_SOURCE[0]}")/.."
```

Comment:
```suggestion
SCRIPT_ROOT="$(dirname "${BASH_SOURCE[0]}")/.."
```

this is not the script root. Let's either rename this to `REPO_ROOT`, or remove the `/..` from this variable definition